src/main/kotlin/no/digdir/fdk/searchservice/service/SearchService.kt

```diff
@@ -28,11 +29,31 @@ class SearchService(
 
     private fun SearchOperation.toElasticQuery(searchTypes: List<SearchType>?): Query {
         val builder = NativeQuery.builder()
-        if (!query.isNullOrBlank()) builder.addFieldsQuery(fields, query)
+
         builder.addFilters(filters, searchTypes)
+
+        if (sort != null) {
+            builder.withSort { sortBuilder ->
+                sortBuilder.field { fieldBuilder ->
+                    fieldBuilder.field(sort.sortField()).order(sort.sortDirection())
+                }
+            }
+        }
```

Comment:
Bare for å holde det likt de andre to:
```suggestion
        if (sort != null) builder.addSorting(sort)
```